.github/workflows/ci.yml

```diff
@@ -54,6 +54,22 @@ jobs:
       - name: Lint code for consistent style
         run: bin/rubocop -f github
 
+  lint_templates:
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: .ruby-version
+          bundler-cache: true
+
+      - name: Lint templates for consistent style
+        run: ./bin/erblint ./app/**/*.erb
```

Comment:
@zachgoll I merged two tasks into one. I prefer calling `./bin/erblint ./app/**/*.erb` because of explicitness, but `--lint-all` also makes sense, updated.